environments/enikshay-reference/app-processes.yml

```diff
@@ -0,0 +1,64 @@
+celery_processes:
+  enikshay-reference-proxy-0:
+    beat: {}
+    celery_periodic:
+      concurrency: 4
+    async_restore_queue,background_queue,case_rule_queue,celery,export_download_queue:
+      concurrency: 1
+    email_queue,reminder_case_update_queue:
+      concurrency: 1
+    reminder_rule_queue,reminder_queue,repeat_record_queue,saved_exports_queue:
+      concurrency: 1
+    sms_queue,submission_reprocessing_queue,ucr_indicator_queue,ucr_queue:
+      concurrency: 1
+    flower: {}
+pillows:
+  enikshay-reference-proxy-0:
+    kafka-ucr-main:
+      num_processes: 1
+#    kafka-ucr-static:
+#      num_processes: 1
+#    AppDbChangeFeedPillow:
+#      num_processes: 1
+#    ApplicationToElasticsearchPillow:
+#      num_processes: 1
+#    CacheInvalidatePillow:
+#      num_processes: 1
+#    CaseSearchToElasticsearchPillow:
+#      num_processes: 1
+#    CaseToElasticsearchPillow:
+#      num_processes: 1
+#    DefaultChangeFeedPillow:
+#      num_processes: 1
+#    DomainDbKafkaPillow:
+#      num_processes: 1
+#    FormSubmissionMetadataTrackerPillow:
+#      num_processes: 1
+#    GeographyFluffPillow:
+#      num_processes: 1
+#    GroupPillow:
+#      num_processes: 1
+#    GroupToUserPillow:
+#      num_processes: 1
+#    KafkaDomainPillow:
+#      num_processes: 1
+#    LedgerToElasticsearchPillow:
+#      num_processes: 1
+#    ReportCaseToElasticsearchPillow:
+#      num_processes: 1
+#    ReportXFormToElasticsearchPillow:
+#      num_processes: 1
+#    SqlSMSPillow:
+#      num_processes: 1
+#    UnknownUsersPillow:
+#      num_processes: 1
+#    UpdateUserSyncHistoryPillow:
+#      num_processes: 1
+#    UserCacheInvalidatePillow:
+#      num_processes: 1
+#    UserGroupsDbKafkaPillow:
+#      num_processes: 1
+#    UserPillow:
+#      num_processes: 1
+#    XFormToElasticsearchPillow:
+#      num_processes: 1
```

Comment:
Are you leaving this in here as a way of documenting that we're intentionally not running them?

Reply:
this was all auto-generated by `commcare-cloud-bootstrap`, but yeah, mostly leaving it as documentation.

Reply:
Yeah, that's commented out in the template file because it only gets used for provable full stack deploy (before this pr anyway), and having a bunch of pillows leads to the memory getting filled up on the `t2.micro` aws instances we use. And they aren't necessary for proving that the deploy worked, so they are commented out. Because this env is only for reference, I'm assuming that it doesn't matter that they're commented out here too